docs/esp8266/quickref.rst

```diff
@@ -159,6 +159,8 @@ Use the ``machine.ADC`` class::
 SPI bus
 -------
 
+See the :ref:`machine.SPI <machine.SPI>` class::
```

Comment:
Not going to help users, as reference doesn't contain information for esp8266. Perfect example of "don't do too many changes at once". It's one thing to markup existing references. It's completely different matter to add new content.